cmd/root.go

```diff
@@ -44,11 +47,19 @@ var (
 	out = colorable.NewColorableStdout()
 )
 
+type FlagError struct{ err error }
```

Comment:
Do we need it to be exported?

Reply:
Indeed, this type does not need to be exported. make the private when your release the next version or i have a new pr? 🤦

Reply:
Thank you Jayson! I made a few updates on this. Made additional changes. 
Please take a peek when I update next.